examples/nextjs/middleware.ts

```diff
@@ -1,9 +1,13 @@
-import { withMiddlewareAuth } from '@supabase/auth-helpers-nextjs/dist/middleware';
+import { withMiddlewareAuth } from '@supabase/auth-helpers-nextjs/middleware';
```

Comment:
I'm not sure if this is going to work, because this will lead to a warning about `jose`. 
```
warn  - ../../node_modules/.pnpm/jose@4.8.1/node_modules/jose/dist/browser/runtime/env.js
A Node.js API is used (process.versions at line: 6) which is not supported in the Edge Runtime.
Learn more: https://nextjs.org/docs/api-reference/edge-runtime
```
This is the reason why we have `/dist/` in the import path.

Reply:
with the dist import I think there may be something odd about the import path too, as I'm hitting `Module '"@supabase/auth-helpers-nextjs/dist/middleware"' has no exported member 'withMiddlewareAuth'.ts(2305)`

Reply:
ah, I just saw that I added an [import configuration](https://github.com/supabase-community/auth-helpers/pull/170/files#diff-e5237f683dda3354b835c7c7c94b9759db2c743d4ba94d47d7f8b8e0b2bfb442R30-R49) for that to the package.json (this PR got really stale for me over the weeks). How are you getting that error?

Reply:
By just running `pnpm dev --filter=@example/nextjs` then opening the site in the browser and you will see the error in the terminal.

Reply:
> with the dist import I think there may be something odd about the import path too, as I'm hitting `Module '"@supabase/auth-helpers-nextjs/dist/middleware"' has no exported member 'withMiddlewareAuth'.ts(2305)`

You will only get this if you leave the [import configuration](https://github.com/supabase-community/auth-helpers/pull/170/files#diff-e5237f683dda3354b835c7c7c94b9759db2c743d4ba94d47d7f8b8e0b2bfb442R30-R49) in, but if you remove that it is no longer there.